### PR TITLE
ENG-2349 add data snapshot page

### PR DESCRIPTION
**Pull request by go-bears**
adding data vue page
port over text
merged in new kv-code-block component
added DataPage to secondary menu
added DataPage to routes.js

### CODE REVIEW

**Comment by mcstover** [comment]
In other news... I'll look into adding the callout styles globally at a later date.

File: src/pages/Build/DataPage.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<template>
+	<www-page>
+		<developer-secondary-menu slot="secondary" />
+		<div class="row page-content">
+			<div class="small-12 columns">
+				<h1>Data Snapshots</h1>
+				<div class="callout secondary">
+					<strong class="show-for-medium-up">Important Note:</strong>
```
Comment (ksekhri) [inline src/pages/Build/DataPage.vue]: Can we add some margin

Comment (go-bears) [reply src/pages/Build/DataPage.vue]: added .5rem to the bottom. Thanks for 👀 !

**Comment by go-bears** [comment]
Thanks @mcstover ! And did the merge locally and pushed the branch back up.

**Comment by BoulderBrains** [comment]
👀

**Comment by go-bears** [comment]
i see you seeing @BoulderBrains !  🎄

**Pull request merged**